web/styles/popovers.css

```diff
@@ -1015,18 +1015,14 @@ ul {
 
         .dropdown-widget-button {
             /* Override the default border-radius to properly align
-            the button corners with `stream_header_colorblock`. */
+            the button corners */
             border-radius: 1px 4px 4px 1px !important;
```

Comment:
If you read the original comment carefully, we only overrode the default border radius because of the color block, so now the border radius rule should be removed (notice the unnecessarily sharp left side borders of the stream picker).